content/concept-xrp-ledger-intro.md

```diff
@@ -0,0 +1,88 @@
+
+**Note:** Non-XRP currencies issued in the XRP Ledger _can_ be frozen. For more information, see the [Freeze documentation](concept-freeze.html).
+
+Furthermore, the XRP Ledger's system of trusted validators uses a small amount of human interaction to achieve better distribution of authority than other decentralized systems. Fully-automated systems for reaching consensus from an unknown set of participants are vulnerable to concentrations of voting power. For example, Bitcoin mining is disproportionately concentrated in places with cheap electricity. As Ripple curates a list of distinct validators operated by different entities in different jurisdictions, the XRP Ledger can become more resistant to censorship and outside pressures than proof-of-work mining. For more information on Ripple's plan to decentralize the recommended set of validators, see the  [Decentralization Strategy Update](https://ripple.com/dev-blog/decentralization-strategy-update/).
+
+
+## Fast, Efficient Consensus Algorithm
+[Fast, Efficient Consensus Algorithm]: #fast-efficient-consensus-algorithm
+
+The XRP Ledger's biggest difference from most cryptocurrencies is that it uses a unique consensus algorithm that does not waste time and energy on "mining", the way Bitcoin, Etherium, and almost all other such systems do. Instead of "proof of work" or even "proof of stake", The XRP Ledger's consensus algorithm uses a system where every participant has an overlapping set of "trusted validators" and those trusted validators efficiently agree on which transactions happen in what order. A single Bitcoin transaction wastes more electricity than a family home in the USA uses in an entire day, and confirming the transaction takes hours. A single XRP transaction uses a negligible amount of electricity, and takes 4 or 5 seconds to confirm. Furthermore, each new "ledger version" in the XRP Ledger (the equivalent of a "block") contains the full current state of all balances, so a server can synchronize with the network in minutes instead of spending hours downloading and re-processing the full transaction history.
+
+For more information on how the XRP Ledger's consensus algorithm works, see [The XRP Ledger Consensus Process](concept-consensus.html). For background on why the XRP Ledger uses a consensus algorithm, see [Reaching Consensus In the XRP Ledger](concept-reaching-consensus.html).
```

Comment:
"why the XRP Ledger uses a consensus algorithm" - maybe you meant "why the XRP Ledger uses **this** consensus algorithm"? It is quite clear why some consensus algorithm is being used...